### PR TITLE
mtu: ability to change running cluster's mtu setting

### PR DESCRIPTION
**Pull request by msherif1234**
Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>
Initial investigation about how MTU change can be implemented for ovn-kubernetes

### CODE REVIEW

File: enhancements/network/cluster-mtu-modification.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- Node's physical NIC's MTU will be handled by the customer.
+- there will be networking outage while MTU changes takes effect, this is
+  will be known, expected and documeneted.
+
+## Proposal
+
+The following diagram shows all places where MTU update need to take effect
+
+![Alt text](./ovn-k8s-node-MTU.png?raw=true)
```
Comment (kedark3) [inline enhancements/network/cluster-mtu-modification.md:43]: I think the diagram is not linked correctly @msherif1234

Comment (msherif1234) [reply enhancements/network/cluster-mtu-modification.md]: you need to select view file to see the diagram

Comment (kedark3) [reply enhancements/network/cluster-mtu-modification.md]: Hmm weird. I thought Rich diff should show it too. NVM.

**Comment by kedark3** [comment]
@msherif1234 do we want to refresh this issue as suggested by the bot in the above comment?

**Pull request closed without merging**